pkg/controllers/ipaddressallocation/ipaddressallocation_controller.go

```diff
@@ -109,8 +105,14 @@ func (r *IPAddressAllocationReconciler) Reconcile(ctx context.Context, req ctrl.
 	log.Info("reconciling IPAddressAllocation CR", "IPAddressAllocation", req.NamespacedName)
 	metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerSyncTotal, MetricResType)
 	if err := r.Client.Get(ctx, req.NamespacedName, obj); err != nil {
-		log.Error(err, "unable to fetch IPAddressAllocation CR", "req", req.NamespacedName)
-		return resultNormal, client.IgnoreNotFound(err)
+		if client.IgnoreNotFound(err) == nil {
```

Comment:
Use `apierrors.IsNotFound` here as other controller for a better understanding?

Reply:
ok